hjson.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+language: hjson
+filename: learnhjson.hjson
+contributors:
+  - ["MrTeferi", "https://github.com/MrTeferi"]
+lang: en
+---
+
+HJSON is an attempt to make [JSON](https://learnxinyminutes.com/docs/json/) more human readable.
+
+HJSON is a syntax extension to JSON. 
```

Comment:
```suggestion
Hjson is a syntax extension to JSON. 
```

Reply:
Alrighty I think it should be good to go now!